### DISCUSSION TITLE
[Feature] Auto archive images in specific folders

### DISCUSSION THREAD

**Discussion by SpiderQubit** [immich-app/immich#14962] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have some folders that sync and upload that i do not want in my timeline and would like to autoarchive them based on folder name. 
Like an option on folders to auto archive every new upload to the folder to keep them from appearing in the timeline.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1673
